src/popup/popupInterfacesSOAP.js

```
@@ -0,0 +1,66 @@
+                document.body.appendChild(a);
+                a.click();
+                document.body.removeChild(a);
+                URL.revokeObjectURL(url);
+                console.log(`Successfully downloaded ${filename}`);
+              } catch (error) {
+                console.error("Error downloading file:", error);
+              }
+            }
+          }
+        );
+      }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)